dist/src/main/config/broker.standalone.yaml.template

```diff
@@ -1152,6 +1152,18 @@
           # decision is evaluated. If the cache is full, the least used DRG gets evicted.
           # drgCacheCapacity: 1000
 
+        # jobs:
+          # Allows to configure the Job Timeout Trigger's batch limit. This is the number of jobs
+          # that could me barked as timed-out in a single trigger run.
+          # This setting can also be set using the environment variable ZEEBE_BROKER_EXPERIMENTAL_ENGINE_JOBS_TIMEOUTTRIGGERBATCHLIMIT
+          # timeoutTriggerBatchLimit: 0x7fffffff
```

Comment:
It adds a decent amount of complexity but since it's already implemented now and it's nice to have for fairness, I'd suggest to keep it now.

:thought_balloon: In general I think it doesn't make much sense to keep these configuration flags as experimental; I don't see how and when we'd decide to promote them to stable. This isn't something we need to change in this PR though.

Reply:
@oleschoenburg @megglos I'd like to extract this functionality and apply it to all checkers to achieve fairness and flow within the periodically executed scheduled tasks.

I imagine having a single config option applied to all checkers equally rather than individual controls. Together with metrics to support tweaking this config for different usages.

While this solution uses a number of resulting records appended to the log in a batch, an alternative would be based on execution time. I'm not sure yet which is better, or whether both might be needed at some point.

Anyways, I instructed @mustafadagher to go for this solution and take the same approach as the Message TTL Checker as it resolves the bug in a safe way and controllable way.